monitoring/monitoring.go

```diff
@@ -68,32 +64,46 @@ type AlertSender interface {
 	SendAlert(alert cloudprotocol.AlertItem)
 }
 
+// Sender sends alerts to the cloud.
+type MonitoringSender interface {
+	SendMonitoringData(monitoringData cloudprotocol.MonitoringData) (err error)
+}
+
 // TrafficMonitoring interface to get network traffic.
 type TrafficMonitoring interface {
 	GetSystemTraffic() (inputTraffic, outputTraffic uint64, err error)
 	GetInstanceTraffic(instanceID string) (inputTraffic, outputTraffic uint64, err error)
 }
 
+// Monitoring configuration for system monitoring.
+type Monitoring struct {
+	MaxOfflineMessages int               `json:"maxOfflineMessages"`
+	Disabled           bool              `json:"disabled"`
```

Comment:
parameter disabled not necessary

Reply:
I guess it is. We will create or not monitoring instance based on this parameter.

Reply:
removed